### ISSUE TITLE
Test failures since version 0.13.2: `NameError: name 'dirty_equals' is not defined`

### ISSUE THREAD

**Issue by danielhollas** [15r10nk/inline-snapshot#118]
Our CI started failing after 0.13.2 release with

```
    def use_valid_old_values(old_value, new_value):
    
>       if isinstance(new_value, dirty_equals.DirtyEquals):
E       NameError: name 'dirty_equals' is not defined

.venv/lib64/python3.12/site-packages/inline_snapshot/_inline_snapshot.py:285: NameError
```

**Comment by 15r10nk** [comment]
thank you for the report, this is fixed in 0.13.3

**Comment by danielhollas** [comment]
Cheers, thank you! 

I was quite confused since `dirty_equals` is mentioned only once in the documentation.
I am wondering to make things clearer if it would make sense to introduce a new optional extras groups with this optional dependency?
